src/components/autocomplete/js/autocompleteController.js

```diff
@@ -208,6 +208,17 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
   }
 
   /**
+   * Find the target for the wrap element which determines menu width
+   * @returns {*}
+   */
+  function getWrapTarget() {
+    for (var element = $element; element.length; element = element.parent()) {
```

Comment:
I'm wondering if there is a better way to accomplish this using a directive. Perhaps the `md-autocomplete-wrap-override` could be a full directive that searches for child autocomplete's (perhaps using some combination of `.querySelector()` and `.controller()`) and register with them as the wrapper so that we don't have to traverse the entire tree of parents all the way back up the DOM.

@aperl You think you could give this a shot?